### ISSUE TITLE
How to add the loss function in the TVM

### ISSUE THREAD

**Issue by Arthur-Shi** [dmlc/tvm#579]
I am trying to do a faster Rcnn by TVM.

And I didnot find any loss function in TVM as far as now.

How can I add the loss function in the TVM?

**Comment by masahi** [comment]
Object detection with nnvm + tvm would be very interesting.
I think you would need some kind of reduction to compute your loss.

tvm has a tutorial on writing reduction [here](http://docs.tvmlang.org/tutorials/language/reduction.html#sphx-glr-tutorials-language-reduction-py)
topi has softmax compute built in, see [here](https://github.com/dmlc/tvm/blob/master/topi/python/topi/nn/softmax.py).

**Comment by Arthur-Shi** [comment]
Thank you so much! @masahi
